docs/book/start/development.md

```diff
@@ -20,7 +20,7 @@ cargo install bpf-linker
 ```
 
 If you are running **macos, or linux on any other architecture**, you need to
-install LLVM 15 first, then install the linker with:
+install LLVM 16 first (for example, with `brew install llvm`), then install the linker with:
```

Comment:
please reflow this, it looks to be the longest line in the file now.

Reply:
done in [3615722](https://github.com/aya-rs/book/pull/126/commits/36157229f2f10027d519f8458c46ed7f0afb8d46)